rdf/rdf12/rdf-semantics/index.html

```diff
@@ -606,7 +606,7 @@ <h2>
         </dt>
         <dd inlist='true' property='mf:entry' resource='../rdf-semantics#opaque-iri-control' typeof='mf:PositiveEntailmentTest'>
           <div property='rdfs:comment'>
-            <p>Check that owl:sameAs works as expected, to ensure that opaque-iri does not pass spuriously.</p>
+            <p>Check that owl:sameAs works as expected, was to ensure that opaque-iri does not pass spuriously.</p>
           </div>
```

Comment:
Should we delete this test as owl:sameAs is not part of RDF? Or maybe list it somehow as a special case?